crnk-core/src/main/java/io/crnk/core/resource/links/DefaultLink.java

```diff
@@ -0,0 +1,92 @@
+package io.crnk.core.resource.links;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+public class DefaultLink implements Link {
```

Comment:
currently wondering whether we should have an interface and default implementation or not. For all the other objects so far we did not have one. But unsure.

Reply:
Right. I've created an interface to follow the pattern of ResourceList and DefaultResourceList.
If also you have time to look at PR #656 that would be great. I need it for a project and currently I'm building a jar file of crnk-core instead of importing through Gradle.